src/pages/tutorials/working-with-feature-layers.md

```diff
@@ -0,0 +1,181 @@
+
+```JavaScript
+var portlandHeritageTrees = L.esri.featureLayer({url: 'https://services.arcgis.com/rOo16HdIMeOBI4Mb/arcgis/rest/services/Heritage_Trees_Portland/FeatureServer/0'}).addTo(map);
+```
+
+If we save and refresh our map, we should see many blue markers on the map, which are the features from the service.  Leaflet is applying a default style to the features.
+
+> Its helpful to cross reference help documents to double-check syntax and learn more about what is possible.  Check out the [Esri Leaflet documentation](http://esri.github.io/esri-leaflet/api-reference/layers/feature-layer.html "Esri Feature Layer") for more information about additional feature layer constructor options.
+
+##### Add Esri Leaflet Renderers Plugin
+
+If we want to use the symbology set when a service was published, we'll need to add the [Esri Leaflet Renderers](https://github.com/Esri/esri-leaflet-renderers "Esri Leaflet Renderers") helper library.  We'll add a `<script>` for this plugin in the `<head>` element on the page.
+
+```xml
+<!-- Load Esri Leaflet Renderers from CDN -->
+<script src="https://cdn.jsdelivr.net/leaflet.esri.renderers/1.0.1/esri-leaflet-renderers.js"></script>  
+```  
+
+If we save and refresh our map, we'll now see the trees displayed as red circles.  It is also possible to define a custom symbology for the features.
+
+> Do we want to provide links to the examples for custom styling?
+
+> http://esri.github.io/esri-leaflet/examples/styling-feature-layer-points.html
+
+> http://esri.github.io/esri-leaflet/examples/styling-feature-layer-polylines.html
+
+> http://esri.github.io/esri-leaflet/examples/styling-feature-layer-polygons.html
+
+
+##### Add a Popup to Feature Layer
+
+Now we will add a popup to our feature layer.  The popup can contain both static text and dynamic text from the fields in the service.  We'll use the `onEachFeature` method to create the popup.
+
+First, we will create a variable for the popup content, using Leaflet's [Utility Template](http://leafletjs.com/reference.html#util-template 'Leaflet Utility Template').  We place the name of fields from the service in  curly brackets **{ }**, and add `feature.properties` at the end of the `L.Util.template` to access the fields.  Second, we will bind the popup to the feature layer.
+
+```JavaScript
+ var portlandHeritageTrees = L.esri.featureLayer({
+    url: 'https://services.arcgis.com/rOo16HdIMeOBI4Mb/arcgis/rest/services/Heritage_Trees_Portland/FeatureServer/0',
+    onEachFeature: function(feature,layer) {
+        var popupContent = L.Util.template('<h3>{COMMON_NAM}</h3><hr /><p>This tree is located at {ADDRESS} and its scientific name is {SCIENTIFIC}.', feature.properties);
+        
+        layer.bindPopup(popupContent);        
+    }
+}).addTo(map);
```

Comment:
i'd recommend just passing a function to `bindPopup()` (and skip calling `onEachFeature()`).

1. its less code
2. its more performant, because the function is only called when someone clicks on a feature.

live demo:
http://esri.github.io/esri-leaflet/examples/feature-layer-popups.html